.github/workflows/delivery/docker/buildpack.yml

```diff
@@ -0,0 +1,16 @@
+go:
+  targets:
+    - ./cmd/pack
+
+  build:
+    # The go.build.flags property allows you to override the default build
+    # flags when compiling your program.
+    flags:
+      - -buildmode=default
+      - -tags=paketo
```

Comment:
These flags are not necessary if you don't need them for a specific reason. We show them in docs as examples of what you might want to override, but you don't NEED them.

Reply:
After experimenting, it seems like I needed the `buildmode=default` (otherwise I got [this](https://github.com/dfreilich/pack/runs/1156366130?check_suite_focus=true) error), but not tags

Reply:
Seems like that is caused the the `${LDFLAGS}` env var that is not interpolated. It looks like the build flags you include are trying to pass some flags to the external linker, `gcc` in this case, and you haven't specified what those are. Do you really need to set `-extldflags`? And if so, can you not hard-code those values?

Reply:
@jromero looks like you added that here: https://github.com/buildpacks/pack/commit/968445647b9b99d995a718208706fdfab5723f9e#diff-b67911656ef5d18c4ae36cb6741b7965. Any chance you remember why?

Reply:
This was a feature request by @ttpcodes, the contributor/collaborator, of the Arch Linux delivery flow. It shouldn't be necessary for this use case since it's a controlled build environment.

Reply:
I'll remove it then.